src/plugins/workspace/server/saved_objects/workspace_id_consumer_wrapper.ts

```diff
@@ -63,8 +64,19 @@ export class WorkspaceIdConsumerWrapper {
           this.formatWorkspaceIdParams(wrapperOptions.request, options)
         ),
       delete: wrapperOptions.client.delete,
-      find: (options: SavedObjectsFindOptions) =>
-        wrapperOptions.client.find(this.formatWorkspaceIdParams(wrapperOptions.request, options)),
+      find: (options: SavedObjectsFindOptions) => {
```

Comment:
The logic used to be in saved objects management page, any reason why we change to this wrapper?

Reply:
I need a central place to put this special logic, and this consumer seems to the place for workspace id related.